samcli/commands/pipeline/bootstrap/cli.py

```diff
@@ -0,0 +1,221 @@
+"""
+CLI command for "pipeline bootstrap", which sets up the require pipeline infrastructure resources
+"""
+import os
+from typing import Any, cast, Dict, List, Optional
+
+import click
+
+from samcli.cli.cli_config_file import configuration_option, TomlProvider
+from samcli.cli.context import get_cmd_names
+from samcli.cli.main import pass_context, common_options, aws_creds_options, print_cmdline_args
+from samcli.lib.config.samconfig import SamConfig
+from samcli.lib.pipeline.bootstrap.stage import Stage
+from samcli.lib.telemetry.metric import track_command
+from samcli.lib.utils.version_checker import check_newer_version
+from .guided_context import GuidedContext
+
+SHORT_HELP = "Sets up infrastructure resources for AWS SAM CI/CD pipelines."
+
+HELP_TEXT = """Sets up the following infrastructure resources for AWS SAM CI/CD pipelines:
+\n\t - Pipeline user(IAM User) with AccessKeyId and SecretAccessKey credentials to be shared with the CI/CD provider
+\n\t - Pipeline Execution Role(IAM Role) that is assumed by the Pipeline user to obtain access to the AWS account
+\n\t - CloudFormation Execution Role(IAM Role) that is assumed by CloudFormation to deploy the SAM application
+\n\t - Artifacts bucket(S3 bucket) to store the sam build artifacts
+\n\t - ECR repo for the container images of Lambda functions having PackageType property set to Image
```

Comment:
Terminology consistent with AWS docs, and added IAM/S3 to the term itself as they're newly introduced in this context:

```suggestion
\n\t - Pipeline IAM user with access key ID and secret access key credentials to be shared with the CI/CD provider
\n\t - Pipeline execution IAM role assumed by the pipeline user to obtain access to the AWS account
\n\t - CloudFormation execution IAM role assumed by CloudFormation to deploy the AWS SAM application
\n\t - Artifacts S3 bucket to store the AWS SAM build artifacts
\n\t - Optionally, an ECR repository to store container image Lambda deployment packages
```